### PR TITLE
update zh-CN translation

### PR DESCRIPTION
**Pull request by lychichem**

### CODE REVIEW

File: main/fdm_zh.po [hunk 1]
```diff
@@ -3792,15 +3792,15 @@ msgstr "取消"
 
 msgctxt "RenameDownloadFileDialog|"
 msgid "Rename file"
-msgstr ""
+msgstr "重命名文件"
```
Comment (LaoshuBaby) [inline main/fdm_zh.po:3795]: 谢谢你！找了半天找到这里，发现已经有人翻译了，不然就重复造轮子了！Thank you for translated this, I almost re-invent wheel。
图片

**Comment by FreeDownloadManagerTeam** [comment]
Too many conflicts in a language I don't know.

**Comment by lychichem** [comment]
> Too many conflicts in a language I don't know.

Where do you think have conflicts?

**Comment by LaoshuBaby** [comment]
> Too many conflicts in a language I don't know.

Could you describe which kind of problem you meet with?

**Pull request closed without merging**